doc/object_finding.rst

```diff
@@ -132,11 +134,12 @@ finding collapses out to search for objects via
             find_min_max = 1600, 2048
 
 This will only collapse out spectral pixels 1600-2048 when computing the 1d SNR
-vs spatial position vector. The best way to choose these pixels is to run pypeit
+vs spatial position vector. The best way to choose these pixels is to run PypeIt
 without it set. Then run :ref:`pypeit_show_2dspec` to view the sky-subtracted
 image and decide which pixels to use for object finding. Then re-run ``PypeIt``.
 
 .. todo::
```

Comment:
It seems like this TODO is not necessary here. The manual extraction is already discussed in extraction.rst

Reply:
Removed!